tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -94,6 +94,12 @@ Verify Documentation Link HTTP Status Code
     ${links}=  Get RHODS Documentation Links From Dashboard
     Check External Links Status     links=${links}
 
+Search and Verify GPU Items Appears In Resources Page
+    [Tags]  ODS-1226
+    Click Link  Resources
+    Sleep   5
+    Input Text  xpath://input[@class="pf-c-search-input__text-input"]   GPU
```

Comment:
You could wrap up these 3 lines in a Keyword called `Search Item In Resource Section` which takes the search term as argument and put this keyword in the Resource file for the dashboard